src/backend/find/settings.py

```diff
@@ -415,6 +478,30 @@ class Development(Base):
 
     USE_SWAGGER = True
 
+    LOGGING = values.DictValue(
+        {
+            "version": 1,
+            "disable_existing_loggers": False,
+            "handlers": {
+                "console": {
+                    "class": "logging.StreamHandler",
+                },
+            },
+            "root": {
+                "handlers": ["console"],
+                "level": "DEBUG",
+                "propagate": True,
+            },
+            "loggers": {
+                "find": {
+                    "handlers": ["console"],
+                    "level": "DEBUG",
+                    "propagate": True,
+                },
+            },
+        }
+    )
```

Comment:
This needs to be moved in the Base configuration (you can have a look at docs settings to do the same)